test/helpers/kubectl.go

```diff
@@ -291,6 +291,12 @@ func CreateKubectl(vmName string, log *logrus.Entry) (k *Kubectl) {
 		defaultHelmOptions["global.registry"] = config.CiliumTestConfig.Registry + "/cilium"
 	}
 
+	// Make sure the namespace Cilium uses exists.
+	ciliumNamespace := GetCiliumNamespace(GetCurrentIntegration())
```

Comment:
Lets use `CiliumNamespace` var from `cons.go` instead of calling this func (nothing really wrong with that, but I think we should use the var for consistency). Will push out a PR which fixes it in this file.

Reply:
fixed